sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -187,6 +189,65 @@ def hpat_pandas_stringmethods_{methodname}_impl(self{methodparams}):
 """
 
 
+@overload_method(StringMethodsType, 'find')
+def hpat_pandas_stringmethods_find(self, sub, start=0, end=None):
+    """
+    Pandas Series method :meth:`pandas.core.strings.StringMethods.find()` implementation.
+
+    Note: Unicode type of list elements are supported only. Numpy.NaN is not supported as elements.
+
+    .. only:: developer
+
+    Test: python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_find
+
+    Parameters
+    ----------
+    self: :class:`pandas.core.strings.StringMethods`
+        input arg
+    sub: :obj:`str`
+        Substring being searched
+    start: :obj:`int`
+        Left edge index
+        *unsupported*
+    end: :obj:`int`
+        Right edge index
+        *unsupported*
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` object
+    """
+
+    ty_checker = TypeChecker('Method find().')
+    ty_checker.check(self, StringMethodsType)
+
+    if not isinstance(sub, (StringLiteral, UnicodeType)):
+        ty_checker.raise_exc(sub, 'str', 'sub')
+
+    accepted_types = (Integer, IntegerLiteral, NoneType, Omitted)
```

Comment:
If I'm not mistaken IntegerLiteral is inherited from Integer, so just `(Integer, NoneType, Omitted)` would be enough. But it is better to check

Reply:
Also I'm not sure if it is better to explicitly add `IntegerLiteral` as supported type or not. On one hand it makes the code more clear, on another, it is confusing that in some places it is specified and in another is not.

Reply:
You are right. `IntegerLiteral` is inherited of `Integer`.